proto/one/enterprise/data/post_filtering_column_definition.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+import "google/protobuf/wrappers.proto";
+
+option csharp_namespace ="ONE.Models.CSharp.Enterprise.Data";
+
+message PostFilteringColumnDefinition {
+
+    // Name of field.
+    string field = 1;
+
+    // Optional displayWidth.
+    google.protobuf.Int32Value displayWidth = 2;
+
+    // Hidden flag.
+    google.protobuf.BoolValue hidden = 3;
+
+    // Sort type.
+    google.protobuf.StringValue sort = 4;
```

Comment:
@KentMiller , this the column definition seemed missing spreadsheet column twin reference `id` which we have it in dataset definition version 0 and version 1, and we have a story on migrate version 0/1 dataset definition to version 2.

Not a blocker to this PR, just a remind of possible near change.